From 0e604a195fc0cf9e8eb132e1b357ec4fbd9c0e89 Mon Sep 17 00:00:00 2001
From: Santiago Rossi <santiago_rho_psi@hotmail.com>
Date: Mon, 16 Apr 2012 21:41:09 -0300
Subject: [PATCH 32/40] 4.4.10 Cuenca de Sholazar: El gorloc enfadado

Fix for quest "The Angry Gorloc" from Oracles/Frenzyheart quest chain
---
 .../2012_04_16_111_world_the_angry_gorloc.sql      |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)
 create mode 100644 sql/updates/world/reanemu/2012_04_16_111_world_the_angry_gorloc.sql

diff --git a/sql/updates/world/reanemu/2012_04_16_111_world_the_angry_gorloc.sql b/sql/updates/world/reanemu/2012_04_16_111_world_the_angry_gorloc.sql
new file mode 100644
index 0000000..4ff62d0
--- /dev/null
+++ b/sql/updates/world/reanemu/2012_04_16_111_world_the_angry_gorloc.sql
@@ -0,0 +1,4 @@
+-- The Angry Gorloc
+DELETE FROM `spell_area` WHERE `spell`=54057;
+INSERT INTO `spell_area` (`spell`,`area`,`quest_start`,`quest_start_active`,`quest_end`,`aura_spell`,`racemask`,`gender`,`autocast`) VALUES
+(54057,4297,12578,1,12578,0,0,2,1);
\ No newline at end of file
-- 
1.7.8.msysgit.0

